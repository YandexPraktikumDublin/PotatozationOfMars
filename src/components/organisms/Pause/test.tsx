import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'

import { Pause } from '.'

describe('<Pause />', () => {
  it('should renders correct <Pause />', () => {
    const wrapper = shallow(<Pause />)

    expect(toJSON(wrapper)).toMatchSnapshot()
  })
})
