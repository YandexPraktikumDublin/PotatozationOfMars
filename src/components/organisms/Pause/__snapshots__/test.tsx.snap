// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Pause /> should renders correct <Pause /> 1`] = `
<button
  className="ml-4"
>
  <img
    alt="Back home"
    src={[Function]}
  />
</button>
`;
